README.md

```diff
@@ -47,7 +47,7 @@ clients are not hardcoding URLs.)
 ## Install
 
 1. [Set up Go](https://golang.org/doc/install) and your `$GOPATH`
-2. `go get -u github.com/letsencrypt/pebble`
+2. `go get -u github.com/letsencrypt/pebble/cmd/pebble`
```

Comment:
Thanks for catching this. Does `go get -u github.com/letsencrypt/pebble/...` work for you on a clean system? I think it would be a better replacement to bring in the pebble-shell and any other cmds we might add in-tree.

Reply:
Definitely. Yep, it works on a clean system. Updating PR.